deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
+
+# export KUBE_NAMESPACE=${ENVIRONMENT:-$DRONE_DEPLOY_TO}
+export KUBE_NAMESPACE="hocs-delta"
+# export KUBE_TOKEN=${KUBE_TOKEN}
+#export VERSION=${VERSION}
+export VERSION="\"1\""
```

Comment:
You shouldn't need to escape the quotation marks here? Either way, allow people to specify the version outside of the script instead of hardcoding it to version 1.